### DISCUSSION TITLE
[FR] Add Login/Website Information to Notes Section

### DISCUSSION THREAD

**Discussion by b3nw** [community-scripts/ProxmoxVE#1095] [Ideas]
Most install scripts post out the url:port and potentially login/reference information to the terminal upon installation. It would be great if that information could be also printed into the Notes section for that container, so its less easily lost.

![image](https://github.com/user-attachments/assets/c5d42dd6-ed9a-4b8c-beab-a037ff85ff6e)

**Comment by Mellowlynx** [comment]
https://github.com/community-scripts/ProxmoxVE/discussions/131#discussioncomment-11191129
